Core/Generators/CSharp/CSharpGenerator.cs

```diff
@@ -46,6 +46,13 @@ public override ValueTask<string> Compile(BebopSchema schema, GeneratorConfig co
                 builder.Indent(indentStep).AppendLine();
             }
 
+            if (Config.EmitBinarySchema)
+            {
+                builder.Append("public const byte[] BEBOP_SCHEMA =");
+                builder.AppendLine(Schema.ToBinary().ConvertToString("new byte[] {", "};"));
+                builder.AppendLine();
+            }
+
             if (Schema.Definitions.Values.Any(d => d is ConstDefinition))
             {
                 builder.AppendLine(ConstantSummary);
```

Comment:
Change this condition so if EmitBinarySchema is true or there are any constants we create the BopConstants class, and put the schema inside this.